spaces-config-service-api/src/main/proto/org/hypertrace/spaces/config/service/v1/space_config_rule.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
+
+option java_multiple_files = true;
+
+package org.hypertrace.spaces.config.service.v1;
+
+
+message SpaceConfigRule {
+  string id = 1;
+  oneof rule_data {
+    AttributeValueData attribute_value_data = 2;
+  }
+}
+
+message AttributeValueData {
```

Comment:
Can we just call it `AttributeData` as it doesn't have any attribute value?

Reply:
Went back and forth on this - the rule type, in plain english, means "generate a space for a span based on the described attribute's value" - so you're right it doesn't have an attribute value, but since this is the configuration for a rule it's describing how to get one. What about `AttributeValueRuleData`?

Reply:
This sounds better